server/internal/adapter/gql/loader_item.go

```diff
@@ -106,22 +108,30 @@ func (c *ItemLoader) FindVersionedItems(ctx context.Context, itemID gqlmodel.ID)
 	return vis, nil
 }
 
-func (c *ItemLoader) FindBySchema(ctx context.Context, schemaID gqlmodel.ID, sort *gqlmodel.ItemSort, p *gqlmodel.Pagination) (*gqlmodel.ItemConnection, error) {
+func (c *ItemLoader) FindByModel(ctx context.Context, modelId gqlmodel.ID, sort *gqlmodel.ItemSort, p *gqlmodel.Pagination) (*gqlmodel.ItemConnection, error) {
 	op := getOperator(ctx)
-	sid, err := gqlmodel.ToID[id.Schema](schemaID)
+	mid, err := gqlmodel.ToID[id.Model](modelId)
+	if err != nil {
+		return nil, err
+	}
+
+	m, err := c.modelUsecase.FindByID(ctx, mid, op)
 	if err != nil {
+		if errors.Is(err, rerror.ErrNotFound) {
+			return nil, nil
+		}
 		return nil, err
 	}
 
-	s, err := c.schemaUsecase.FindByID(ctx, sid, op)
+	s, err := c.schemaUsecase.FindByID(ctx, m.Schema(), op)
 	if err != nil {
 		if errors.Is(err, rerror.ErrNotFound) {
 			return nil, nil
 		}
 		return nil, err
 	}
```

Comment:
they are not needed I think

Reply:
I think that schema is needed for this line of code:
`itm := gqlmodel.ToItem(i, s)`